src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
Yes, that's right -- sorry, I thought @javierluraschi had changes he wanted to incorporate into Ace itself as well.